releasenotes/notes/add-datadog-trace-tag-propagation-f69736976107f1c2.yaml

```diff
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    Add support for Datadog trace tag propagation
```

Comment:
Should the release note explain what new environment variables are introduced with this PR?

Reply:
I don't think this is needed in the release notes. That should be included in documentation for this feature when that lands.